src/flavors/defaultflavor/config.json

```diff
@@ -852,12 +854,29 @@
       },
       {
         "category": "observation",
+        "formId": "observation",
         "includeOnForm": true,
         "includeOnList": true,
         "submitter_editing_supported": true,
         "name": "location_type",
         "datasetSlug": "demo",
         "icon_url": "/static/css/images/markers/marker-observation.png",
+        "geospatialAnalysis": [
```

Comment:
Here are some examples of the kind of configuration that geospatial fields would need. I'm not sure how this impacts backend development, but if necessary I think we could group all pieces of geospatial configuration (like the details of aggregations) into a JSON object rather than model all that explicitly.